shared/reducers/fs.tsx

```diff
@@ -595,6 +595,31 @@ export default function(state: Types.State = initialState, action: FsGen.Actions
             : driverStatus
         )
       )
+    case FsGen.tlfCrStatusChanged:
+      const parsedPath = Constants.parsePath(action.payload.tlfPath)
+      const newState = action.payload.status
+      if (parsedPath.kind === Types.PathKind.Root || parsedPath.kind === Types.PathKind.TlfList) {
+        // This should not happen.
+        return state
+      }
```

Comment:
`in-group-tlf` and `in-team-tlf` also shouldn't happen right? So could probably test for `group-tlf` and `team-tlf`.

Reply:
I only tested for these two because that way `parsedPath` is definitely a type that has a `tlfPath`. I think I'd rather this be permissive and work if given an in-TLF path. WDYT?

Reply:
`tlfPath` is supposed to be just paths to TLFs I think; I'd actually rather be strict here than causing confusion down the road.